src/web/mjs/connectors/Sukima.mjs

```diff
@@ -14,41 +14,51 @@ export default class Sukima extends Connector {
     }
 
     async _getMangas() {
-        let categorys = await this._fetchPOST('/api/book/v1/free/', '{"store":false,"genre":"0"}');
-
         let uri;
-        let pages;
-        let mangas = [];
+        let mangaList = [];
+        const pageContent = await this._fetchPOST('/api/v1/search/', JSON.stringify({
```

Comment:
May simplify the existing `_fetchPOST()` method to do the `JSON.stringify` of the body within the method itself, so it does not have to be done every time the method is called.